orttraining/orttraining/python/training/onnxblock/engine/training_module.py

```diff
@@ -0,0 +1,112 @@
+import numpy as np
+
+from onnxruntime.capi import _pybind_state as C
+from onnxruntime.capi.onnxruntime_pybind11_state import OrtValueVector
+
+
+class TrainingModule:
```

Comment:
perhaps this can be called just `Module` since it will be imported from onnxruntime.training.